applications/connecting_applications_to_services/understanding-service-binding-operator.adoc

```diff
@@ -31,26 +31,18 @@ The {servicebinding-title} consists of a controller and an accompanying custom r
 
 As a result, the {servicebinding-title} enables workloads to use backing services or external services by automatically collecting and sharing binding data with the workloads. The process involves making the backing service bindable and binding the workload and the service together.
 
-
 [id="making-an-operator-managed-backing-service-bindable"]
 === Making an Operator-managed backing service bindable
 To make a service bindable, as an Operator provider you need to expose the binding data required by workloads to bind with the services provided by the Operator. You can provide the binding data either as annotations or as descriptors in the CRD of the Operator that manages the backing service.
 
-
 [id="binding-a-workload-together-with-a-backing-service"]
 === Binding a workload together with a backing service
 By using the {servicebinding-title}, as an application developer, you need to declare the intent of establishing a binding connection. You must create a `Service Binding` CR  that references the backing service. This action triggers the {servicebinding-title} to project the exposed binding data into the workload. The {servicebinding-title} receives the declared intent and binds the workload together with the backing service.
 
 The CRD of the {servicebinding-title} supports the following APIs:
 
 * *Service Binding* with the `binding.operators.coreos.com` API group.
-* *Service Binding (Spec API Tech Preview)* with the `servicebinding.io` API group.
-+
-[IMPORTANT]
-====
-*Service Binding (Spec API Tech Preview)* with the `servicebinding.io` API group is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
-For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
-====
+* *Service Binding (Spec API)* with the `servicebinding.io` API group.
```

Comment:
It's unclear why these instances of service binding are bold and capitalized. 
Most instances of "service binding" in our docs look like this:
```
Results for "Service Binding" in "openshift-docs/*.adoc"

/home/rdlugyhe/openshift-docs/architecture/service_catalog/index.adoc:
  82:  the service bindings must be removed first. Deleting the service bindings is
  82:  the service bindings must be removed first. Deleting the service bindings is
  84:  that references the service binding being deleted.
  86:  Once all the service bindings are removed, the service instance may be deleted.
  89:  If a project or namespace containing service bindings and service instances is
  99:  If you delete a service binding used by a deployment, you must also remove any
  202:  Service Binding::
  203:  A _service binding_ is a link between a service instance and an application.
  234:  service broker when using either service bindings or service instances. The only
  237:  in the service binding request. For parameters that need more security, place
  240:  .Example Service Binding Resource Referencing a Secret


/home/rdlugyhe/openshift-docs/architecture/service_catalog/service_catalog_cli.adoc:
  349:  Create the service binding using the command:
  369:  ===== View service binding details
  370:  . To view service binding details:
  416:  To delete service catalog related resources, you need to unbind service bindings
  420:  === Deleting service bindings
  422:  . To delete all service bindings associated with a service instance:
  446:  .  Verify that all service bindings are deleted:
  464:  Running this command deletes all service bindings for the instance. For deleting individual bindings from
```

Reply:
How about:
```
The {servicebinding-title} CRD supports the `binding.operators.coreos.com` and  `servicebinding.io` API groups.
```

Reply:
I'm not seeing any instances of "Spec API" in the repo.

Reply:
@rolfedh In the OCP UI, as soon as you install the SBO, these are the APIs that appear and this is how they look. I prefer to retain it as is.